CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.3.11
+  - Fix crash that occurs when receiving string input that cannot be coerced to UTF-8 (such as BLOB data) [#291](https://github.com/logstash-plugins/logstash-input-jdbc/pull/291)
```

Comment:
Super nit: Action word should be in the [past tense]( https://github.com/elastic/logstash/blob/master/CONTRIBUTING.md#what-is-the-format-of-the-changelog-entries)
Feel free to merge after changing that